src/NovelRT/CMakeLists.txt

```diff
@@ -167,14 +167,32 @@ if(WIN32)
   add_custom_command(TARGET Engine POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy_directory
       ${CMAKE_BINARY_DIR}/thirdparty
       $<TARGET_FILE_DIR:Engine>)
-  file(GLOB libs RELATIVE ${CMAKE_BINARY_DIR}/thirdparty ${CMAKE_BINARY_DIR}/thirdparty/*.dll)
+  file(GLOB libs ABSOLUTE ${CMAKE_BINARY_DIR}/thirdparty/*.dll)
   set_target_properties(Engine PROPERTIES RESOURCE "${libs}")
 endif()
 if(APPLE)
   add_custom_command(TARGET Engine POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy_directory
           ${CMAKE_BINARY_DIR}/thirdparty
           $<TARGET_FILE_DIR:Engine>)
-  file(GLOB libs RELATIVE ${CMAKE_BINARY_DIR}/thirdparty ${CMAKE_BINARY_DIR}/thirdparty/*.dylib)
+  file(GLOB libs ABSOLUTE ${CMAKE_BINARY_DIR}/thirdparty/*.dylib)
   set_target_properties(Engine PROPERTIES RESOURCE "${libs}")
 endif()
 
+install(
+  TARGETS Engine
+  EXPORT NovelRT
+  LIBRARY DESTINATION lib
+  RESOURCE DESTINATION bin
+)
+#Workaround due to hierarchy destruction when installing by TARGETS
+install(
+  DIRECTORY ${CMAKE_SOURCE_DIR}/include/NovelRT/
+  DESTINATION include/NovelRT
+  FILES_MATCHING PATTERN "*.h"
+)
+#Only done so that shaders are always copied
+install(
```

Comment:
This should be in the resources CMakeLists

Reply:
Moved in the latest version (i believe)